source/scripting/Vehicle.cs

```diff
@@ -361,6 +283,10 @@ public sealed class Vehicle : Entity
 	{
 		public Vehicle(int handle) : base(handle)
 		{
+			Doors = new VehicleDoorCollection(this);
+			Mods = new VehicleModCollection(this);
+			Wheels = new VehicleWheelCollection(this);
+			Windows = new VehicleWindowCollection(this);
```

Comment:
Not a fan on initializing all these in the constructor, most of the time someone creates a vehicle class they aren't interested in the doors/mods/wheels/windows. better to initialize them in their respective getters. Also with how the INativeValue is currently set up, the constructor for Vehicle wont be called if its being returned from a native so calling the getters would result in a null reference error